packages/gatsby-theme-docs/src/layouts/internals/layout-design-utils.ts

```diff
@@ -0,0 +1,81 @@
+import { designSystem } from '@commercetools-docs/ui-kit';
+import { contentLayoutConfig } from './layout-design-config';
+
+/**
+ * getPageLayoutGridStyles utility is a helper function
+ * to share layout styles across multiple components
+ *
+ * @param gridRows string of locally defined grid rows
+ * @param isReleaseNotesPage boolean to determine if the page is a release notes page
+ * @param allowWideContentLayout boolean to determine if the page should allow wide content layout
+ * @returns
+ */
+export const getPageLayoutGridStyles = ({
```

Comment:
This is the single entry point for layout components to get their grid styling from. We can avoid layout drift by reducing the duplicated code